src/plugins/condition/components/Condition.vue

```diff
@@ -263,12 +255,15 @@ export default {
         destroy() {
         },
         removeCondition(ev) {
-            this.$emit('removeCondition', this.conditionIdentifier);
+            this.$emit('removeCondition', {
+                condition: this.condition,
+                index: this.conditionIndex
+            });
```

Comment:
I think we only need to pass index for condition to be removed. I think in the receiving function call condition is retrieved via id.

Reply:
Done.